age_checker.info.yml

```diff
@@ -1,6 +1,9 @@
 name: Age Gate
-description: This module will provide an age gate for the sites.
-package: Age Gate
-core: 8.x
 type: module
-configure: age_checker.settings
+description: "This module will provide an age gate for the sites."
+package: Age Gate
+version: '8.x-0.x-dev'
+core: '8.x'
+project: 'age_checker'
+datestamp: 1472644262
```

Comment:
Do we need to add datestamp or version explicitly, I think these are added by drupal.org.
Not sure on this